week-1/errors/2.js

```diff
@@ -1,5 +1,8 @@
 // Currently trying to print the string "I was born in Bolton" but it isn't working...
 // what's the error ?
 
-console.log(`I was born in ${cityOfBirth}`);
+
 const cityOfBirth = "Bolton";
+console.log(`I was born in ${cityOfBirth}`);
+
+//porque a linha do console estava vindo antes entao primeiro tenho que declarar e depois rodar
```

Comment:
I translated this, and it is correct - but you should try translating it back into English too. This is so that you can practice your technical English language skills if you apply for a job in English.